.editorconfig

```diff
@@ -0,0 +1,16 @@
+# This file is for unifying the coding style for different editors and IDEs
+# editorconfig.org
+root = true
+
+[*]
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+indent_style = space
+indent_size = 4
```

Comment:
Looks like we're currently using 4 everywhere except less, in a future version of the UI we'll be switching to 2 spaces, but we can make that change later.